network_diffusion/tpn/functions.py

```diff
@@ -0,0 +1,50 @@
+"""Script with functions for driver node selections."""
```

Comment:
Since in this script we're not handling Temporal Networks any more, and Multilayer Graphs are now a backbone of these functions, I'd move it to `network_diffusion.mln` module and name it `driver_nodes(actors?).py`

Reply:
Yes, I agree.

Reply:
It's up to you :) But since they're related with a particular functionality, maybe it's better to store them in a separate file